src/app/home/home.page.html

```diff
@@ -11,7 +11,15 @@
     >Avviare il server in locale, e disabilitare momentaneamente i middleware di
     authenticazione e authorizzazione</ion-label
   >
-  <ion-tab-button tab="signup" [routerLink]="'/signup'">
-    <ion-label>let's start</ion-label>
-  </ion-tab-button>
+  <ion-tabs>
+    <ion-tab-bar slot="bottom">
```

Comment:
Rimuovere le tabs alla fine del refactoring. Non è detto che utilizzeremo le tabs, potremmo usare un menu. E in ogni caso faremo affidamento ai componenti che sta creando gabriele.

Reply:
Ho messo le tabs temporaneamente perché dovevo switchare in qualche modo tra login e signup, non per altro.

Reply:
Lo so, per questo devi rimuoverle solamente alla fine.